CHANGES/928.contrib.rst

```diff
@@ -0,0 +1 @@
+Added tests to have full code coverage of the ``_viewbaseset_richcmp`` function -- by :user:`skinnyBat`
```

Comment:
We need a small punctuation correction — a trailing period. I also backed in a more specific path to the function:
```suggestion
Added tests to have full code coverage of the
``multidict._multidict_base._viewbaseset_richcmp()`` function
-- by :user:`skinnyBat`.
```